### ISSUE TITLE
When will the ckpts and training scripts be released?

### ISSUE THREAD

**Issue by gigasurgeon** [VlSomers/prtreid#1]
Thank you for the work. When will the ckpts and training scripts be released? I wanted to train it on basketball dataset. And infer it on clips > 2 minutes.

**Comment by VlSomers** [comment]
Hi @gigasurgeon there are already weights available within https://github.com/SoccerNet/sn-gamestate, can you check there and let me know if you found what you were looking for?

**Comment by VlSomers** [comment]
Hi @gigasurgeon, did you manage to run the codebase?

**Comment by gigasurgeon** [comment]
Hi @VlSomers , I couldn't make the sn-gamestate repo that you had mentioned work for basketball. I did some preliminary work with clip embeddings though.

**Comment by jeremyfix** [comment]
@VlSomers  do the checkpoints available on zenodo (from these urls https://github.com/SoccerNet/sn-gamestate/blob/057dd144a982e00576f8ffb45bdd00c0f614c549/sn_gamestate/reid/prtreid_api.py#L96-L105) able to produce the body part masks you are showing in the paper ? 

I tried using the checkpoint available on the soccernet baseline but it does not seem to produce masks on the parts; I may not be using it the right way though.

**Comment by VlSomers** [comment]
HI @jeremyfix, unfortunately, the version of PRTreID that was used in sn-gamestate repo was no trained with the multi part head producing body part masks, because it was easier for the soccernet challenge to have a simple baseline producing a single embeddings and not multi embeddings as in the PRTreID paper. If you are really interested in this model, you can still retrain it though! :)

**Comment by jeremyfix** [comment]
Thank you for you super quick feedback . I understand the point. Thank you again !
